apps/api/v2/.prettierrc.js

```diff
@@ -2,5 +2,6 @@ const rootConfig = require("../../../packages/config/prettier-preset");
 
 module.exports = {
   ...rootConfig,
+  importOrder: ["^./instrument", ...rootConfig.importOrder],
```

Comment:
According to the Sentry Nest docs @ https://docs.sentry.io/platforms/javascript/guides/nestjs/#use - the import of instrument.ts should be the first thing that happens in the app.ts file; before any other imports. This rule override ensures that import "./instrument"; indeed happens first thing.